README.md

```diff
@@ -34,21 +36,24 @@ Almost everything.
 - RelationPage: updating `availableToMentor` and `needsMentoring` status doesn't work
 - About screen is basically About dialog. It's very simple, more info should be added.
 
-Occasional UI bugs may occur – in this case, please create an issue.
+Occasional UI bugs may occur – in this case, please [file an issue](https://github.com/anitab-org/mentorship-flutter/issues/new/choose).
 
 ### Overview
 
-- App _tries_ to follow Clean Architecure guidelines. Logic is separated into 4 layers:
+The architecture of the app isn't top-notch, but currently it does it job quite well. If you have an idea
```

Comment:
I think this sentence is not needed, its a bit self deprecating. For example, what is top notch architecture!? There is always room for improvement in a project. Backend and Android aren't perfect and could have some refactoring, and that is fine :)
Let's keep this objective. if someone wishes to open an issue to improve the app, they'll do it anyways :) If you feel something is clearly missing you can even create the issue yourself, for someone to solve.

Reply:
Yeah, this makes sense :)